addons/medical/ACE_Settings.hpp

```diff
@@ -59,6 +59,16 @@ class ACE_Settings {
         typeName = "SCALAR";
         values[] = {"Disabled", "50/50", "Enabled"};
     };
+    class GVAR(autoWakeUp) {
+        typeName = "BOOL";
+        value = 1; // default 0, left 1 for testing
+        isClientSettable = 0;
+    };
+    class GVAR(autoWakeUpTime) {
+        typeName = "SCALAR";
```

Comment:
Also please set the value to `-1` as default. And I'd suggest adjusting the setting name to:
`maxUnconsciousTime`